src/lib.rs

```diff
@@ -3,6 +3,16 @@
 // <LICENSE-MIT or http://opensource.org/licenses/MIT>, at your option.
 // All files in the project carrying such notice may not be copied, modified, or distributed
 // except according to those terms.
+//! Raw FFI bindings to all of Windows API.
+//! # Features
+//! Every submodule of this crate's modules lies behind a compiler feature of the same name.
+//! The only exceptions are [`winapi::shared::guiddef`](shared/guiddef/) which requires no feature,
```

Comment:
There is still a feature for this module, but the module doesn't use it due to the top level `Class`/`Interface` traits needing `GUID`. I'd prefer to leave out the bit about `guiddef`.